types/random-string-gen/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for random-string-gen 1.1
+// Project: https://github.com/MadeByRaymond/randomStringGenerator
```

Comment:
what's the project home?
https://github.com/yuhenabc/random#readme

Reply:
https://www.npmjs.com/package/random-string-gen
https://github.com/MadeByRaymond/randomStringGenerator#readme

Reply:
thx, I've read package name wrongly